sklearn/neighbors/_base.py

```diff
@@ -141,7 +141,7 @@ def _get_weights(dist, weights):
         )
 
 
-def _is_sorted_by_data(graph):
+def _is_sorted_by_row_values(graph):
```

Comment:
If we want to make reviewing easier for other reviewers, this function name change can be done in a separate PR.